compass/landice/tests/ismip6_run_ais/run_model.py

```diff
@@ -0,0 +1,156 @@
+import os
+from compass.model import run_model, make_graph_file
+from compass.step import Step
+
+
+class RunModel(Step):
+    """
+    A step for performing forward MALI runs as part of Antarctica test cases.
+
+    Attributes
+    ----------
+    mesh_file : str
+        The name of the mesh file being used
+
+    mesh_type : str
+        The resolution or mesh type of the test case
+    """
+
+    def __init__(self, test_case, mesh_type, name='run_model'):
+        """
+        Create a new test case
+
+        Parameters
+        ----------
+        mesh_file : str
+            The name of the mesh file being used
+
+        mesh_type : {'high', 'mid'}
+            The resolution or mesh type of the test case
+        """
+
+        self.mesh_file = None
+        self.mesh_type = mesh_type
+
+        super().__init__(test_case=test_case, name=name)
+
+    def setup(self, velo_solver="FO"):
+        config = self.config
+        section = config['ismip6_run_ais']
+        base_path_mali = section.get('base_path_mali')
+        calving_law = section.get('calving_law')
+        damage = section.get('damage')
+        procs = section.get('procs')
+
+        if calving_law not in ['none', 'floating', 'eigencalving',
+                               'specified_calving_velocity',
+                               'von_Mises_stress',
+                               'damagecalving', 'ismip6_retreat']:
+            raise ValueError("Value of calving_law must be one of {'none', "
+                             "'floating', "
+                             "'eigencalving', 'specified_calving_velocity', "
+                             "'von_Mises_stress', 'damagecalving', "
+                             "'ismip6_retreat'}")
+
+        self.add_input_file(filename='albany_input.yaml',
+                            package='compass.landice.tests.ismip6_run_ais',
+                            copy=True)
+
+        self.add_model_as_input()
+
+        self.add_output_file(filename='output.nc')
+
+        # Todo: confirm the number of cores needed for the high mesh
+        # We estimate that 200-1000 cells should be allocated for one core
+        res_param = {
+                'high': {'mesh_file': 'Antarctic_1to10km.nc',
+                         'mask_file': 'regionMask_AIS_1to10km.nc',
+                         'cores': 25000,
+                         'min_cores': 5000},
+                'mid': {'mesh_file': 'Antarctic_8to80km_20220407.nc',
+                        'mask_file': 'regionMask_AIS_8to10km.nc',
```

Comment:
As we discussed in our meeting, let's choose a naming convention for regions masks that is  meshname.regionMask.nc

Reply:
It might be worth giving the masks a date stamp, too.  We have decided to add new regions in the past in MPAS-Analysis and it became rather awkward to keep things backwards compatible without a date stamp.